defaults/main.yml

```diff
@@ -39,3 +39,5 @@ osbs_build_json_dir: /usr/share/osbs
 osbs_sources_command: fedpkg sources
 osbs_vendor: Fedora Project
 osbs_nodeselector: ''
+osbs_all_node_selector_labels:
```

Comment:
Can we rename this to something more intuitive? Maybe `osbs_managed_node_labels` clarifies the intent that these will be managed by ansible and may be removed from the node if needed.

BTW, `..._node_selector_labels` seems redundant. In this context, these should just be `..._node_labels`.

Reply:
ok

Reply:
I echo that. A *node selector* is used by a thing that wants to find a node to use, and is expressed as a set of conditions based on the values of *node labels*, i.e. the labels on a node.